### PR TITLE
add arc-view

### PR DESCRIPTION
**Pull request by bharatnpti**

### CODE REVIEW

File: init/install_services.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ kubectl create secret generic lmos-runtime --from-literal=OPENAI_API_KEY="$OPENA
 
 # Install lmos-runtime chart
 helm upgrade --install lmos-runtime oci://ghcr.io/lmos-ai/lmos-runtime-chart \
- --version 0.6.0 \
+ --version 0.0.11-SNAPSHOT \
```
Comment (kaikreuzer) [inline init/install_services.sh:35]: from a 0.6.0 release to a 0.0.11-SNAPSHOT?

File: init/install_services.sh [hunk 2]
```diff
@@ -53,6 +53,7 @@ kubectl create secret generic openai-secrets \
 
 helm upgrade --install weather-agent oci://ghcr.io/lmos-ai/weather-agent-chart --version 1.0.7-SNAPSHOT
 helm upgrade --install news-agent oci://ghcr.io/lmos-ai/news-agent-chart --version 1.0.7-SNAPSHOT
+helm upgrade --install arc-view-runtime-web oci://ghcr.io/lmos-ai/arc-view-runtime-web-chart --version 0.1.0
```
Comment (kaikreuzer) [inline init/install_services.sh:56]: I wouldn't add this to the "#Install agents" section, but rather create a new separate section for it.

**Review by pschwager — COMMENTED** [review]
Could you add the new endpoint to the readme?

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
lgtm, just 2 small questions/comments.

File: init/install_services.sh (hunk 1 above)
Comment (bharatnpti) [reply init/install_services.sh:35]: we had multiple snapshot versions from 0.6 to 0.11 but not released
0.0.11-SNAPSHOT has temp fix for CORS, do not want to enable it by default

Comment (kaikreuzer) [reply init/install_services.sh:35]: So I would assume you mean a `0.11.0-SNAPHOT` version instead of a `0.0.11-SNAPSHOT`?

**Review by kaikreuzer — APPROVED** [review]
Thanks!

**Pull request merged**